docs/src/using-turing/guide.md

```diff
@@ -42,7 +42,12 @@ end
 ```
 
 
-Note: As a sanity check, the expectation of `s` is 49/24 (2.04166666...) and the expectation of `m` is 7/6 (1.16666666...).
+Note: As a sanity check, the prior expectation of `s²` is 3 (= theta/(alpha - 1), where theta is 3 and alpha is 2) and the prior expectation of `m` is 0. This can be easily checked using `Prior`:
```

Comment:
Maybe another alternative could be
```suggestion
Note: As a sanity check, the prior expectation of `s²` is `mean(InverseGamma(2, 3)) = 3/(2 - 1) = 3` and the prior expectation of `m` is 0. This can be easily checked using `Prior`:
```

Reply:
Yes, you are right, that seems good too.